### PR TITLE
Add ongoing and latest failed builds to BuildInfo

### PR DESCRIPTION
**Pull request by dkurepa**
image

### CODE REVIEW

File: src/ProductConstructionService/ProductConstructionService.BarViz/Code/Helpers/DateFormatter.cs [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+{
+    public static string FormatDate(DateTimeOffset date)
+    {
+        var now = DateTimeOffset.Now;
+        var timeFormat = "HH:mm";
+        var dateFormat = "yyyy-MM-dd";
+
+        if (date.Date == now.Date)
+        {
+            return $"Today at {date.ToString(timeFormat)}";
```
Comment (premun) [inline src/ProductConstructionService/ProductConstructionService.BarViz/Code/Helpers/DateFormatter.cs:16]: Why not just put `date.ToString(timeFormat)` in a variable?

File: src/ProductConstructionService/ProductConstructionService.Client/Generated/AzDo.cs [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+            Client = client ?? throw new ArgumentNullException(nameof(client));
+        }
+
+        public ProductConstructionServiceApi Client { get; }
+
+        partial void HandleFailedRequest(RestApiException ex);
+
+        partial void HandleFailedGetBuildStatusRequest(RestApiException ex);
+
+        public async Task<IImmutableList<Models.AzDoBuild>> GetBuildStatusAsync(
```
Comment (premun) [inline src/ProductConstructionService/ProductConstructionService.Client/Generated/AzDo.cs:43]: Can you turn these into `List<>`?

File: src/ProductConstructionService/ProductConstructionService.BarViz/Code/Helpers/DateFormatter.cs (hunk 1 above)
Comment (dkurepa) [reply src/ProductConstructionService/ProductConstructionService.BarViz/Code/Helpers/DateFormatter.cs]: done

File: src/ProductConstructionService/ProductConstructionService.Client/Generated/AzDo.cs (hunk 2 above)
Comment (dkurepa) [reply src/ProductConstructionService/ProductConstructionService.Client/Generated/AzDo.cs]: done

**Review by premun — APPROVED** [review]

**Pull request merged**